bool is_calibrated
bool flight_mode
